src/components/Filters.tsx

```diff
@@ -1,90 +1,109 @@
-import React, {useState} from 'react'
+import React from 'react'
 import {
   Paper,
   Icon,
-  ExpansionPanelSummary,
-  ExpansionPanelDetails,
-  Radio,
+  Box,
   Chip,
   Typography,
-  FormControlLabel,
-  RadioGroup,
-  Button,
   Avatar,
-  Box,
-  Snackbar
+  Button,
+  ButtonGroup,
+  Hidden,
+  Checkbox,
 } from '@material-ui/core'
-import {Alert} from '@material-ui/lab'
 import {useFilterStyles} from '../classes'
 import {ManagedQuestionJSON} from '../types'
-// import {Link} from 'react-router-dom'
 import DoneIcon from '@material-ui/icons/Done';
 import TagIcon from '@material-ui/icons/LocalOfferOutlined';
 import IntegrationIcon from '@material-ui/icons/BuildOutlined';
 import FilterListIcon from '@material-ui/icons/FilterListRounded';
-import OpenInNewIcon from '@material-ui/icons/OpenInNew';
-import copy from 'clipboard-copy'
+import CategoryIcon from '@material-ui/icons/CategoryOutlined';
+import { useWindowSize } from "@reach/window-size";
 
 interface Props {
   managedQuestions: ManagedQuestionJSON;
   integrationClicked: Function;
-  integration: string;
+  integrations: string[];
   allTags: string[];
   tagCheckClicked: Function;
   tags: string[];
-  search: string;
-  clear: Function;
+  filter: string;
+  setFilterLogic: Function;
+  allCategories: string[];
+  categories: string[];
+  setCategories: Function;
 }
 
 const Filters = (props: Props) => {
   const classes = useFilterStyles()
-  const [copied, setCopied] = useState(false)
+  const windowSize = useWindowSize()
 
   return (
-    <Paper className={classes.root}>
-      <ExpansionPanelSummary>
-        <Icon>
-          <FilterListIcon/>
-        </Icon>
-        <Typography variant='h6'>Filters</Typography>
-      </ExpansionPanelSummary>
-      <ExpansionPanelSummary>
-        <Icon>
-          <IntegrationIcon/>
-        </Icon>
-        <Typography variant='subtitle1'>Integrations</Typography>
-      </ExpansionPanelSummary>
-      <ExpansionPanelDetails className={classes.notFlex}>
-        {[...Object.keys(props.managedQuestions.integrations), 'none'].map((integration: any, index: number) => (
-          <RadioGroup color='secondary' key={index} className={classes.notFlex} value={props.integration} onChange={() => props.integrationClicked(integration)}>
-            <FormControlLabel value={integration} control={<Radio/>} label={integration}/>
-          </RadioGroup>
-        ))}
-      </ExpansionPanelDetails>
-
-      <ExpansionPanelSummary>
-        <Icon>
-          <TagIcon/>
-        </Icon>
-        <Typography variant='subtitle1'>Tags</Typography>
-      </ExpansionPanelSummary>
-      <ExpansionPanelDetails className={classes.flexWrap}>
-        <Box mb={2}>
+    <Hidden>
+      <Paper elevation={0} className={windowSize.width > 750 ? classes.root : classes.smallRoot}>
```

Comment:
Same thing as before. It's fine for now but we'll have to revisit for future work.